cli/src/semgrep/constants.py

```diff
@@ -32,13 +32,13 @@
 
 class EngineType(Enum):
     OSS = auto()
-    PRO = auto()
-    INTERPROC = auto()
-    INTERFILE = auto()
+    PRO_LANG = auto()
+    PRO_INTRAFILE = auto()
+    PRO_DEEP = auto()
```

Comment:
Or `PRO_INTERFILE` ? Somebody had observed that "intrafile" and "interfile" were very similar. 🤷‍♂️ Another option could be `PRO_FULL` since the idea is that it provides the "full experience".